docs/shortcuts.md

```diff
@@ -38,8 +38,9 @@
 
 ### View
 
-| Function   | Mac |   Other    |
-| ---------- | :-: | :--------: |
-| Zoom in    | ⌘+  | `Ctrl + +` |
-| Zoom out   | ⌘-  | `Ctrl + -` |
-| Zoom reset | ⌘0  | `Ctrl + 0` |
+| Function    | Mac |   Other    |
+| ----------- | :-: | :--------: |
+| Zoom in     | ⌘+  | `Ctrl + +` |
+| Zoom out    | ⌘-  | `Ctrl + -` |
+| Zoom reset  | ⌘0  | `Ctrl + 0` |
+| Full Screen |  F  |    `F`     |
```

Comment:
revert that change.. we'll delete this file (#1220) in favour of https://howto.excalidraw.com/shortcuts/ (where I already added the shortcut)